docs/library/uasyncio.rst

```diff
@@ -93,8 +96,8 @@ class Task
 
 .. method:: Task.cancel()
 
-    Cancel the task by injecting a ``CancelledError`` into it.  The task may
-    or may not ignore this exception.
+    Cancel the task. Tasks may intercept cancellation (for example to perform
```

Comment:
I think it's important to keep the first sentence as-is, ie telling that cancellation is achieved by injecting a `uasyncio.CancelledError`.  Otherwise how would one know this?